frontend/src/components/Dashboard.tsx

```
@@ -0,0 +1,152 @@
+        
+        {!editing ? (
+          <div>
+            <p><strong>Mobile:</strong> {profile.mobile_number}</p>
+            <p><strong>Username:</strong> {profile.username}</p>
+            <p><strong>Email:</strong> {profile.email || 'Not set'}</p>
+            <p><strong>Name:</strong> {profile.first_name} {profile.last_name}</p>
+            <p><strong>National ID:</strong> {profile.national_id || 'Not set'}</p>
+            <p><strong>Verified:</strong> {profile.is_verified ? 'Yes' : 'No'}</p>
+            <p><strong>Joined:</strong> {new Date(profile.date_joined).toLocaleDateString()}</p>
+            
+            <button onClick={() => setEditing(true)} style={{ marginTop: '10px' }}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)